service/test/agama/registration_test.rb

```diff
@@ -226,25 +236,25 @@
               .to raise_error(Timeout::Error)
           end
 
-          it "does not run the callbacks" do
-            expect(callback).to_not receive(:call)
-
+          it "sets the registration code" do
             expect { subject.register("11112222", email: "test@test.com") }
               .to raise_error(Timeout::Error)
+
+            expect(subject.reg_code).to eq("11112222")
           end
 
-          it "does not set the registration code" do
+          it "does not set the email" do
```

Comment:
hmm, title says does not set, but in test below it is set. This looks confusing to me.